src/index.html

```diff
@@ -6,14 +6,7 @@
     <title>To Do List</title>
 </head>
   <body>
-    <div id="todoDiv">
-      <h1 class="task">Today's To Do</h1>
-      <form id="add"  class="task">
-        <input id="addTask" type="text" name="addTask" value="" placeholder="Add to your list...">
-        <button type="submit" id="enter"></button>
-      </form>
-      <ul id="todoList">
-      </ul>
-    </div>
+    <ul id="todoList">
+    </ul>
```

Comment:
- [ ] **Required:** Kindly add the form even if not implemented at the moment. The wireframe is important for this milestone.